src/main/webapp/assets/modules/current-job/salary-review.con.js

```diff
@@ -141,10 +161,13 @@ techlooper.controller("salaryReviewController", function ($scope, $rootScope, js
       delete $scope.error[modelName];
       var inputValue = $scope.$eval(modelName);
       var notHasValue = ($.type(inputValue) === "array") && (inputValue.length === 0);
-      notHasValue = notHasValue || (inputValue === null);
+      notHasValue = notHasValue || !inputValue;
       notHasValue = notHasValue || (inputValue.length <= 0);
       notHasValue && ($scope.error[modelName] = $rootScope.translate.requiredThisField);
     });
+    $scope.salaryReview.skills.length || ($scope.error.skills = $rootScope.translate.requiredThisField);
```

Comment:
Line is too long.<br>Expected an assignment or function call and instead saw an expression.